functions/exercise/5_findlargestInNumberArray.js

```diff
@@ -8,4 +8,23 @@
 
 
 // Sample function : findLargestNumber([])
-// Expected return value : null
\ No newline at end of file
+// Expected return value : null
+
+
+function findLargestNumber(array) {
+  if (array.length === 1) {
+    return array[0];
+  }
+  if (array.length === 0) {
+    return null;
+  }
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] > array[i + 1]) {
+      return array[i];
+    }
```

Comment:
I dont think this will work it will return the first element it finds that is larger than next items. If you write the loop iteration in details then you will find the mistake